server/src/main/java/org/tbbtalent/server/util/html/JsoupTextExtracterImpl.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Talent Beyond Boundaries.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+package org.tbbtalent.server.util.html;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Safelist;
+import org.springframework.stereotype.Service;
+
+/**
+ * JSoup https://jsoup.org/ implementation of Text Extraction from HTML
+ *
+ * @author Tim Hill
+ */
+@Service
+public class JsoupTextExtracterImpl implements TextExtracter {
+    /**
+     * Given an HTML string extract just the text
+     *
+     * @param html an HTML string
+     * @return the text from the HTML. Tags replaced with space
+     */
+    @Override
+    public String ExtractText(String html) {
```

Comment:
probably just a typo - but best use camel casing for method name here (i.e. extractText)

Reply:
lol, I think you've spotted I'm not a real Java developer. Let's chat about the different approaches with the html strip character filter https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-htmlstrip-charfilter.html and Jsoup text extract before I look to resolve this.

Reply:
Corrected.